example/server/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+
+from polaris.models import Transaction
+
+
+class PolarisUser(models.Model):
+    first_name = models.CharField(max_length=254)
+    last_name = models.CharField(max_length=254)
+    email = models.EmailField(unique=True)
+
+    objects = models.Manager()
+
+    @property
+    def name(self):
+        return " ".join([str(self.first_name), str(self.last_name)])
+
+
+class PolarisUserAccount(models.Model):
```

Comment:
This should be PolarisStellarAccount since it represents a stellar account, not an anchors user account